### PR TITLE
Add import/export statements to Secret server code

### PR DESCRIPTION
**Pull request by lorensr**
For #338

### CODE REVIEW

File: content/security.md [hunk 1]
```diff
@@ -338,16 +338,23 @@ Secret business logic in your app should be located in code that is only loaded
...
 ```
 
 ```js
 // In a file loaded on client and server
+let MMR;
+if (Meteor.isServer) {
+  MMR = require('/imports/server/mmr.js');
+}
+
 const Meteor.users.methods.updateMMR = new ValidatedMethod({
```
Comment (abernix) [inline content/security.md]: How does this even work?

File: content/security.md [hunk 2]
```diff
@@ -338,16 +338,23 @@ Secret business logic in your app should be located in code that is only loaded
...
 }
+
+export default MMR;
 ```
 
 ```js
 // In a file loaded on client and server
+let MMR;
+if (Meteor.isServer) {
+  MMR = require('/imports/server/mmr.js');
```
Comment (abernix) [inline content/security.md]: Would this be an opportunity to use a dynamic `import()`?

Or alternatively, could this just be in the `} else {` below?

```js
require("/imports/server/mmr.js").updateWithSecretAlgorithm(this.userId);
```

File: content/security.md (hunk 1 above)
Comment (lorensr) [reply content/security.md]: Copied from sashko. Tried, and it doesn't 😄

**Comment by lorensr** [comment]
Fixed, thanks! 

Tested here: https://github.com/lorensr/demo-secret-method-code

File: content/security.md (hunk 1 above)
Comment (abernix) [reply content/security.md]: Haha, I was confused by the entire thing... including this which I think is still a problem, even if not directly related to this PRs fix:

```
const Meteor.users.methods.updateMMR = new ValidatedMethod({
```

Comment (abernix) [reply content/security.md]: I mean, I'm positive that's a syntax error, specifically because of the `const` declaration.

**Pull request merged**